### DISCUSSION TITLE
How to add 950 color shade ? Manually or alpha release ?

### DISCUSSION THREAD

**Discussion by Extarys** [skeletonlabs/skeleton#2794] [Q&A] [answered]
In this issue (https://github.com/skeletonlabs/skeleton/issues/1142) the new 950 color shade is ticked... but I'm guessing the changes aren't yet released as stable.

Trying to add `--color-primary-950` throws `Object literal may only specify known properties, and '"--color-primary-950"' does not exist in type 'ThemeProperties'`.

In the meantime, should I simply add the new color shade to my `:root` element manually? I also noticed while writing this that there is a v3 alpha release. Since the project is small, should I give it a try ?

What are your thoughts? :)

**Comment by endigo9740** [comment, marked as answer]
@Extarys the 950 shade did not exist in Tailwind when Skeleton v2 was created. As such we will be introducing support for it in our next major version called Skeleton v3 (aka "Next") - this is what you're referring to as an alpha. This version is available in a VERY EARLY pre-release state and is primarily available for preview and testing at this time. It's not feature complete and does not have feature parity with Skeleton v2 at this time. You can learn more here:

- https://github.com/skeletonlabs/skeleton/discussions/2681

We very strongly recommend against using v3 production at this time. So I'd advise trying to configure your design to utilize up to shade 900 for now. We post regular updates for v3, so feel free to keep up with the status of that over the next couple months.

**Reply by Extarys** [reply]
Hey @endigo9740, thanks for taking the time. You're the expert, therefore I shall try not to use 950 for the time being :)
